### PR TITLE
User can search within a board for a specific keybinding card

### PR DESCRIPTION
**Pull request by kevinram**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    var $card = newCard($parent,matchingBindings[i]);
+    $parent.appendChild($card);
+  }
+  return $parent;
+}
+
+function newCard($parent,binding){
+  var $card = $doc.createElement('div');
+  $card.classList.add('col')
+  for (var field in cardFields){
```
Comment (thebearingedge) [inline main.js]: Avoid using `for...in` to iterate through arrays; it's non-idiomatic and may cause confusion. Use a regular `for` loop instead.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+var bindings = [
+  { name: "Markdown Preview Toggle", category: "Markdown", keybinding: "ctrl-shift-M", desc: "Show Markdwn"},
+  { name: "Markdown Blah", category: "Markdown", keybinding: "ctrl-blah-M", desc: "Blahdown  "},
+  { name: "Open Link", category: "Browser", keybinding: "ctrl-shift-o", desc: "Opens up a HTTP or HTTPS link"}
+];
+var cardFields = ["name","category","keybinding","desc"]
```
Comment (thebearingedge) [inline main.js]: Use a single space after each comma in a comma-separated list:
```js
var cardFields = ["name", "category", "keybinding", "desc"]
```
Also avoid abbreviations. If `"desc"` means `"description"`, write `"description"`.

File: main.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+var $doc = document;
+var $searchBtn = $doc.getElementById('search')
+
+$searchBtn.addEventListener('click', function(){
+  var phrase = $doc.getElementById('phrase').value;
+  var matchedBindings = qryShortcuts(phrase, bindings);
```
Comment (thebearingedge) [inline main.js]: Again, avoid abbreviations. `qryShortcuts -> queryShortcuts`.

**Comment by kevinram** [comment]
Unsure about naming syntax for my array of element names.  Based on your feedback I chose cardProps.  Req feedback.  @thebearingedge

**Comment by thebearingedge** [comment]
@kevinram `cardFields` was ok, but `cardProps` is acceptable too.

File: main.js [hunk 4]
```diff
@@ -1,20 +1,20 @@
 var bindings = [
-  { name: "Markdown Preview Toggle", category: "Markdown", keybinding: "ctrl-shift-M", desc: "Show Markdwn"},
-  { name: "Markdown Blah", category: "Markdown", keybinding: "ctrl-blah-M", desc: "Blahdown  "},
-  { name: "Open Link", category: "Browser", keybinding: "ctrl-shift-o", desc: "Opens up a HTTP or HTTPS link"}
+  { name: "Markdown Preview Toggle", category: "Markdown", keybinding: "ctrl-shift-M", description: "Show Markdwn"},
+  { name: "Markdown Blah", category: "Markdown", keybinding: "ctrl-blah-M", description: "Blahdown  "},
+  { name: "Open Link", category: "Browser", keybinding: "ctrl-shift-o", description: "Opens up a HTTP or HTTPS link"}
 ];
-var cardFields = ["name","category","keybinding","desc"]
+var cardProps = ["name","category","keybinding","desc"]
```
Comment (thebearingedge) [inline main.js]: I meant to put spaces after the commas within a list. This applies to parameter lists and arguments lists.

**Pull request merged**